qiita_db/study.py

```diff
@@ -347,6 +348,59 @@ def create(cls, owner, title, efo, info, investigation=None):
 
         return cls(study_id)
 
+    @classmethod
+    def delete(cls, id_):
+        r"""Deletes the study from the database
+
+        Parameters
+        ----------
+        id_ : integer
+            The object identifier
+
+        Raises
+        ------
+        QiitaDBError
+            If the sample_(id_) table exists means a sample template exists
+        """
+        cls._check_subclass()
+
+        # checking that the id_ exists
+        cls(id_)
```

Comment:
Just to confirm, what you are asking is for me to solve
https://github.com/biocore/qiita/issues/1078 within this PR and use
that function?